docs/rackhd2.0/configuration.rst

```diff
@@ -15,6 +15,7 @@ config.json_
         "apiServerAddress": "172.31.128.1",
         "apiServerPort": 9080,
         "broadcastaddr": "172.31.131.255",
+        "dhcpPollerActive": true,
```

Comment:
Do you want to default enable the DHCP poller? Since we have ARP poller now, so I think we'd better disable DHCP poller defautly. If so, change this to `false` will be better.